mopidy_soundcloud/library.py

```diff
@@ -151,19 +151,19 @@ def lookup(self, uri):
             uri = uri.replace("sc:", "")
             return self.backend.remote.resolve_url(uri)
 
-        if "directory" in uri:
+        if uri == "soundcloud:directory":
             return
```

Comment:
In this case, I think you should return an empty list, rather than `None`.

Reply:
Thanks! I'll update it